low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -45,12 +50,16 @@ def __init__(self):
         self.sysfs.initialize()
         self.sysfs_base_path = self.sysfs.get_sysfs_base_path()
         self.cpu_path = self.sysfs_base_path + CPU_PATH
-        self.server_frus = {
+        hw_resources = {
             'cpu': self.get_cpu_info,
             'platform_sensors': self.get_platform_sensors_info,
             'memory': self.get_mem_info,
-            'fans': self.get_fans_info,
-            'nw_ports': self.get_nw_ports_info
+            'fans': self.get_fans_info
+        }
+        sw_resources = {}
+        self.server_resources = {
```

Comment:
seems we had missed putting this hierarchy too, having hw, sw

Reply:
Now we have. If resource not found in hw or sw, `resource_found` will be False.
```
        if not resource_found:
            raise ResourceMapError(
                errno.EINVAL,
                f"Invalid rpath or health provider doesn't have support for'{rpath}'.")
```

Reply:
Ok